types/react-router-navigation-core/react-router-navigation-core-tests.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import {
+    StyleSheet,
+    Dimensions,
+    View,
+    StyleProp,
+    ViewStyle
+} from "react-native";
+import { TabStack, renderSubView } from "react-router-navigation-core";
```

Comment:
Could you try to narrow this test down to just using these two things? This seems to mostly be testing other react packages.

Reply:
yes I will do, for this test I just copied one of the Lib files which use the core-lib